src/main/flight/pid.h

```diff
@@ -150,12 +150,13 @@ typedef struct pidProfile_s {
     uint8_t abs_control_error_limit;        // Limit to the accumulated error
     uint8_t dterm_filter2_type;             // Filter selection for 2nd dterm
     uint16_t dyn_lpf_dterm_max_hz;
-    uint8_t  dyn_lpf_dterm_idle;
```

Comment:
This requires an increment of the parameter group version, see https://github.com/betaflight/betaflight/blob/master/docs/development/ParameterGroups.md.

Reply:
This increment is still needed.